packages/mdc-menu/mdc-menu.scss

```diff
@@ -63,13 +63,14 @@
   .mdc-list-item {
     @include mdc-rtl-reflexive-box(padding, left, 56px);
   }
-}
 
-.mdc-menu__selection-group-icon {
-  @include mdc-rtl-reflexive-position(left, 16px);
+  // Extra specificity required because list-item graphic needed to increase specificity.
```

Comment:
I don't see `.mdc-list-item__graphic` anywhere on the screenshot test page. How is this related?

Reply:
Nit: Can you update this comment to something like:

```js
// Extra specificity required to override `display` property on `mdc-list-item__graphic`.
```